Psychtoolbox/PsychHardware/EyelinkToolbox/EyelinkBasic/EyelinkBasicLegacy/EyelinkEraseCalTarget.m

```diff
@@ -0,0 +1,4 @@
+function EyelinkEraseCalTarget(el, rect)
```

Comment:
Actual erase code missing.

Reply:
resolved in 33984c0ef6fc9d2a3dc40f30ab9710ad031d9938